additional-data/mdn-docs.json

```
@@ -1404,37 +1409,54 @@
   "link": [
     "Web/HTML/Element/link"
   ],
-  "link-selectors": [],
+  "link-selectors": [
+    "Web/CSS/:link",
+    "Web/CSS/:visited",
+    "Web/CSS/:any-link"
+  ],
   "list-elements": [
     "Web/HTML/Element/ol",
     "Web/HTML/Element/ul",
     "Web/HTML/Element/li"
   ],
-  "list-style": [],
+  "list-style": [
+    "Web/CSS/list-style"
+  ],
   "loading-lazy": [
     "Web/API/HTMLIFrameElement/loading",
     "Web/API/HTMLImageElement/loading"
   ],
-  "local-fonts": [],
+  "local-fonts": [
+    "Web/API/Window/queryLocalFonts"
+  ],
   "localstorage": [
     "Web/API/Web_Storage_API"
   ],
   "location": [
     "Web/API/Document/location"
   ],
-  "logical-assignments": [],
+  "logical-assignments": [
+    "Web/JavaScript/Reference/Operators/Logical_AND",
+    "Web/JavaScript/Reference/Operators/Logical_OR"
+  ],
   "logical-properties": [
     "Web/CSS/CSS_logical_properties_and_values"
   ],
   "long-animation-frames": [
     "Web/API/PerformanceLongAnimationFrameTiming"
   ],
-  "longtasks": [],
-  "magnetometer": [],
+  "longtasks": [
+    "Web/API/PerformanceLongTaskTiming"
+  ],
+  "magnetometer": [
+    "Web/API/Magnetometer"
+  ],
   "main": [
     "Web/HTML/Element/main"
   ],
-  "managed-media-source": [],
+  "managed-media-source": [
+    "Web/API/MediaSource"
```

```suggestion
    "Web/API/Media_Source_Extensions_API"
```

Let's link to the API landing page for now. There isn't a doc for ManagedMediaSource unfortunately. I've filed this issue for it: https://github.com/mdn/mdn/issues/611